README.md

```diff
@@ -45,15 +45,64 @@ strategy:
     exclude:
       - { os: windows-2019, php: "8.1" }
       - { os: windows-2019, php: "8.0" }
-      - { os: windows-2019, php: "7.4" }
-      - { os: windows-2019, php: "7.3" }
+      - { os: windows-2022, php: "7.4" }
+      - { os: windows-2022, php: "7.3" }
       - { os: windows-2022, php: "7.2" }
       - { os: windows-2022, php: "7.1" }
 ```
 
 Currently, `windows-2019` may be used for all PHP versions, although this may
 change in future releases.
 
+### Manually Installing Toolsets
```

Comment:
This was adapted from https://github.com/mongodb/mongo-php-driver/pull/1565. I have no idea how to install VC14 for pre-7.1 builds, as I couldn't find a component ID for that in the [Visual Studio docs](https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2022).

Reply:
Would it make sense to add steps to the action here to conditionally install the toolset for 7.1, 7.2, and 7.3?